plugins/provisioners/ansible/cap/guest/redhat/ansible_install.rb

```diff
@@ -0,0 +1,29 @@
+
+module VagrantPlugins
+  module Ansible
+    module Cap
+      module Guest
+        module RedHat
+          module AnsibleInstall
+
+            # This should work on recent Fedora releases, and any other members of the
```

Comment:
This comment should be fixed as well (_on recent Fedora releases_). Feel free to propose something, otherwise I'll take care of it when merging ;-)